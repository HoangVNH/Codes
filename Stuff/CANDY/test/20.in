10
35
DDDDDDDDDXXXXXVVDDDXXXXXXXDDDDDDDVV
25
XDDDDDDDVVVDDDDDDDDDDVVVV
33
XXXDDDXXXXXVDDDDVVVVDDDDDDDDVVVXX
48
XXXXDDDDDVDDDDVVVVVXXXXXXXDDDVVVVVVVVVXXXXXDXDDD
30
VVXXXXXXDDDXXXXXVVVVVVVVVXXXXX
29
DDVVVVDDDDDVVXXDDDDDVVVVXDDDD
47
VVVVVVVDDDDVVVVVVVDDDXXXDDDDDDDDVVVXXXXXXXXXXXX
21
VVVVVXXXXXXXVVDDDDDXX
40
XXXXXDDDXXXXXXXXXXXXXXXVVVVDDXXXDDVVVVDD
32
XXXXDDDXXXXXXDDXXXXXXXXDVVVVVVVV
