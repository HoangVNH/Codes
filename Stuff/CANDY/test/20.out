34
23
32
47
29
28
45
19
39
30
